Customer Support Procedures/Neo4j-v5.0-migration.md

```diff
@@ -0,0 +1,47 @@
+<!-- omit in toc -->
+# Neo4j migration to v5 guidelines
+
+If you are planning to migrate your Neo4j database from an older version to v5,
+this document will provide you extra details of what you need to take care of in your Linkurious Enterprise project.
+
+<!-- omit in toc -->
+# Table of contents
+- [Pre-requisites and limitations](#pre-requisites-and-limitations)
+- [Issues with cypher queries](#issues-with-cypher-queries)
+- [Migrating databases](#migrating-databases)
+
+# Pre-requisites and limitations
```

Comment:
Did you have the chance to very the impact of the upgrade in LKE? If it will cause the need of a reindex, we should mention it somewhere in this document.

Reply:
Customers using incremental indexing may have to reindex. I'll check that and update the this section accordingly.

Reply:
See https://linkurious.atlassian.net/browse/LKE-7689: a full reindex is needed after migrating to Neo4j v5.

I've added a paragraph about that.